test/dynamo/test_repros.py

```diff
@@ -3874,6 +3874,40 @@ def randint_fn(high, size, out):
         out2 = torch.empty(12, dtype=torch.int32)
         opt_model(17, (12,), out2)
 
+    @requires_cuda
```

Comment:
Why does it have to be a CUDA test? Similar leak will occur with CPU inductor, wouldn't it?

Reply:
@anijain2305 explained that its needed because test uses CUDA API to check memory consumption